### PR TITLE
Adds back cryogenic fluid

### PR DESCRIPTION
**Pull request by Uqualm**
Here you go.

### CODE REVIEW

**Comment by banthebantz** [comment]
thank you

**Comment by tortellinitony** [comment]
Did you add the new files to `hippiestation.dme`, because I don't see changes. I'm not sure how this compiled...

File: hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+/datum/reagent/cryogenic_fluid/on_tick()
+	holder.chem_temp -= 5
+	..()
+
+/datum/reagent/cryogenic_fluid/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1)
+	if(iscarbon(M) && M.stat != DEAD)
+		if(method in list(INGEST,INJECT))
+			M.bodytemperature = max(M.bodytemperature - 50, TCMB)
+			if(show_message)
+				M << "<span class='warning'>You feel like you are freezing from the inside!</span>"
```
Comment (tortellinitony) [inline hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: to_chat

File: hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		else
+			if (reac_volume >= 5)
+				if(show_message)
+					M << "<span class='danger'>You can feel your body freezing up and your metabolism slow, the pain is excruciating!</span>"
```
Comment (tortellinitony) [inline hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: to_chat

File: hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		var/turf/open/O = T
+		if(hotspot)
+			if(O.air)
+				var/datum/gas_mixture/G = O.air
+				G.temperature = 0
+				G.react()
+				qdel(hotspot)
+
+		if(reac_volume >= 6)
+			T.freon_gas_act() //freon in my pocket
```
Comment (tortellinitony) [inline hippiestation/code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm]: Change this to `O.freon_gas_act()`

**Pull request merged**